### PR TITLE
customize map with tile overlay

### PR DESCRIPTION
**Pull request by ochanje210**
now u can customize map

### CODE REVIEW

File: shared/components/map.js [hunk 1]
```diff
@@ -44,6 +44,7 @@ export default class Map extends Component {
   render() {
     return (
       <View style ={styles.container}>
+
```
Comment (lyw0149) [inline shared/components/map.js]: Delete this line.

**Comment by lyw0149** [comment]
Thanks! Go!! goober Go!! LGTN

**Comment by lyw0149** [comment]
Thanks! Go!! goober Go!! LGTM

**Comment by mun625** [comment]
LGTM

**Pull request merged**